resources/wenquanyi/README

```diff
@@ -20,20 +20,22 @@ from wqy-bitmapsong-bdf-1.0.0-RC1.tar.gz
 wenquanyi_cjk_basic_9pt.bdf
 GPLv2+FE
 
+is generated by the following coommand:
```

Comment:
I've added this change to make the English text correspond to the Chinese text below.